### DISCUSSION TITLE
使用antd5.x时，如果要使用antd自带的less变量需要使用convertLegacyToken转一下，请问转了之后的返回值字典在哪里能看呢？

### DISCUSSION THREAD

**Discussion by rurounibatousai** [ant-design/ant-design#39173] [Q&A]
### 以下是我目前能查看的位置;
- ![image](https://user-images.githubusercontent.com/42639431/205195259-e249df62-005e-4ce9-a96a-74a2a552fc27.png)
- ![image](https://user-images.githubusercontent.com/42639431/205195343-f26cbf8b-eb82-4455-b765-6b71e78582cd.png)

**Comment by q269384828** [comment]
run-serve 的时候可以输出到文件吧, `fs.writeFile`, 只看key的话可以有ts声明的, 另外跑起来了在less文件里面都会有的
